src/app/@ansyn/ansyn/app-providers/map-source-providers/open-layers-not-geo-registered-planet-source-provider.ts

```diff
@@ -0,0 +1,9 @@
+import { Injectable } from '@angular/core';
+import { OpenLayerPlanetSourceProvider } from './open-layers-planet-source-provider';
+
+export const OpenLayerNotGeoRegisteredPlanetSourceProviderSourceType = 'NOT-GEO-REGISTERED-PLANET';
```

Comment:
`OpenLayerNotGeoRegisteredPlanetSourceProviderSourceType` so simple

Reply:
Just followed the ongoing naming convention.